Tests/configure_and_test_integration_instances.py

```diff
@@ -634,6 +636,46 @@ def report_tests_status(preupdate_fails, postupdate_fails, preupdate_success, po
     return testing_status
 
 
+def set_marketplace_gcp_bucket_for_build(client, prints_manager, branch_name, ci_build_number):
+    """Sets custom marketplace GCP bucket based on branch name and build number
+
```

Comment:
from the name and the docs I understood that it creates the bucket.
maybe it us better to say it configures the server to use the bucket with the branch name and build number.